worker/src/RTC/RtpStreamSend.cpp

```diff
@@ -494,8 +481,43 @@ namespace RTC
 
 		// Store original packet and some extra info into the retrieved storage item.
 		storageItem->originalPacket = clonedPacket;
-		storageItem->ssrc           = clonedPacket->GetSsrc();
-		storageItem->sequenceNumber = clonedPacket->GetSequenceNumber();
+		storageItem->ssrc           = packet->GetSsrc();
+		storageItem->sequenceNumber = packet->GetSequenceNumber();
+		storageItem->timestamp      = packet->GetTimestamp();
```

Comment:
I remember the timestamp is not modified in `v3` unless I recall it incorrectly, why is it necessary now?

Reply:
It is: It is, it is: https://github.com/versatica/mediasoup/blob/v3/worker/src/RTC/SimulcastConsumer.cpp#L876

Reply:
Hm, but clearly not for all consumer types: https://github.com/versatica/mediasoup/blob/v3/worker/src/RTC/SimpleConsumer.cpp#L277-L279
Why? Is there a problem with using original timestamps?

Reply:
here: https://github.com/versatica/mediasoup/pull/302